spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/cache/CacheProperties.java

```diff
@@ -233,4 +242,72 @@ public void setConfig(Resource config) {
 
 	}
 
+	/**
+	 * Redis-specific cache properties. Properties set will be used as the defaults for
+	 * all Redis caches.
+	 *
+	 * @see org.springframework.data.redis.cache.RedisCacheConfiguration
+	 */
+	public static class Redis {
+
+		/**
+		 * The TTL for entries. By default, entries do not expire, a value of zero
```

Comment:
Configuration key descriptions do not start with `The`, `A`, etc. Please look at [existing keys](https://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html) for inspiration